autorag/schema/metricinput.py

```diff
@@ -1,93 +1,117 @@
 from dataclasses import dataclass
 from typing import Optional, List, Dict, Callable, Any, Union
+from itertools import chain
 
 import numpy as np
 import pandas as pd
+from deepeval.test_case import LLMTestCase
```

Comment:
import deepeval inside the `to_deepeval_testcase` 
Because I don't want deepeval import everytime use `MetricInput`.